README.md

```diff
@@ -4,6 +4,12 @@
 This project allows students to express their coding skills before entering
 CodeU. Students will complete this JSON-lite object and JSON-lite parser.
 
+## DISCLAIMER
+
+CODEU is a program created by Google to develop the skills of future software
+engineers. This project is not an offical Google Product. This project allows
+students to express their coding skills before entering CodeU.
```

Comment:
Be consistent in how you capitalize CodeU (it's CODEU earlier in this paragraph)